types/node/http2.d.ts

```diff
@@ -849,7 +849,7 @@ declare module "http2" {
         ): this;
         addListener(
             event: "goaway",
-            listener: (errorCode: number, lastStreamID: number, opaqueData: Buffer) => void,
+            listener: (errorCode: number, lastStreamID: number, opaqueData?: Buffer) => void,
```

Comment:
LGTM

You have added changes to version for TypeScript <= v4.8. Please add the same changes to `node/ts4.8` folder.

Also it looks like `opaqueData` can be `undefined` for Node.js v18 and v16 too. So please add changes to `node/v16`, `node/v16/ts4.8`, `node/v18`, `node/v18/ts4.8` folders also.